### PR TITLE
fix(multi content config): Fixed problem on closing multi content configuration modal

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**
SD-2417

### CODE REVIEW

File: client/app/scripts/superdesk-desks/aggregate.js [hunk 1]
```diff
@@ -182,6 +183,10 @@
                     return values;
                 };
 
+                scope.isWidget = function() {
+                    return this.type === 'widget' ? true : false;
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/aggregate.js]: The `this.type === 'widget'` part is already enough.

File: client/app/scripts/superdesk-desks/views/aggregate-settings.html [hunk 2]
```diff
@@ -1,72 +1,68 @@
...
+</div>
 
-        <div class="modal-header">
-            <a href="" class="close" ng-click="cancel()"><i class="icon-close-small"></i></a>
-            <h3 translate>Aggregate settings</h3>
-        </div>
+<div class="modal-body" sd-wizard data-name="aggregatesettings" data-current-step="step.current" data-finish="cancel()">
 
-        <div class="modal-body" sd-wizard data-name="aggregatesettings" data-current-step="step.current" data-finish="cancel()">
+    <div sd-wizard-step data-title="{{ 'Desks' | translate }}" data-code="desks">
```
Comment (petrjasek) [inline client/app/scripts/superdesk-desks/views/aggregate-settings.html]: you can use one time binding for translate like `{{ :: 'Desks' | translate }}`

**Comment by ioanpocol** [comment]
:+1:

File: client/app/scripts/superdesk-desks/views/aggregate-settings.html [hunk 3]
```diff
@@ -1,72 +1,68 @@
...
-                    </div>
-                    <div class="modal-footer">
-                        <button id="nextStages" class="btn btn-primary" ng-click="next()" translate>Next</button>
-                    </div>
+    <div sd-wizard-step data-title="{{ :: 'Saved Searches' | translate }}" data-code="searches">
+        <div class="content">
+            <div class="legend" translate>Select saved searches for view</div>
+            <div class="desk" ng-repeat="search in searches">
+                <div class="desk-title">
+                    {{search.name}}
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/aggregate-settings.html]: Does this name change or can we use one-time binding here?

File: client/app/scripts/superdesk-desks/views/aggregate-settings.html [hunk 4]
```diff
@@ -1,72 +1,68 @@
...
+    </div>
 
-                    </div>
-                    <div class="modal-footer">
-                        <button id="nextStages" class="btn btn-primary" ng-click="next()" translate>Next</button>
-                    </div>
+    <div sd-wizard-step data-title="{{ :: 'Saved Searches' | translate }}" data-code="searches">
+        <div class="content">
+            <div class="legend" translate>Select saved searches for view</div>
+            <div class="desk" ng-repeat="search in searches">
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/aggregate-settings.html]: Does this `searches` list get updated? (if so, `track by` should be added)

File: client/app/scripts/superdesk-desks/views/aggregate-settings.html [hunk 5]
```diff
@@ -1,72 +1,68 @@
...
-        <div class="modal-body" sd-wizard data-name="aggregatesettings" data-current-step="step.current" data-finish="cancel()">
+    <div sd-wizard-step data-title="{{ :: 'Desks' | translate }}" data-code="desks">
+        <div class="content">
 
-            <div sd-wizard-step data-title="{{ 'Desks' | translate }}" data-code="desks">
-                    <div class="content">
+            <div class="legend" translate>Select desks for view</div>
+            <div class="desk" ng-repeat="desk in desks">
+                <div class="desk-title">
+                    {{desk.name}}
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/aggregate-settings.html]: One-time binding? (if the name of the desk does not change in this view)

File: client/app/scripts/superdesk-desks/views/aggregate-settings.html [hunk 6]
```diff
@@ -1,72 +1,68 @@
...
+                    <span sd-switch class="pull-right" ng-model="editGroups[desk._id].selected" ng-click="setDeskInfo(desk._id)"></span>
+                </div>
+                <ul class="stages clearfix" ng-if="editGroups[desk._id].selected">
+                    <li ng-repeat="stage in deskStages[desk._id]">
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/aggregate-settings.html]: Can stages be added to a desk while this view is active? If yes, we should add `track by` here, if not, ignore this comment. :)

**Comment by plamut** [comment]
Wait, this has been merged already?

**Pull request merged**